packages/perseus/src/widgets/interactive-graphs/graphs/linear-system.tsx

```diff
@@ -28,12 +31,70 @@ const LinearSystemGraph = (props: LinearSystemGraphProps) => {
     const {dispatch} = props;
     const {coords: lines} = props.graphState;
 
+    const {strings, locale} = usePerseusI18n();
+    const id = React.useId();
+
+    const linesAriaInfo = lines.map((line, i) => {
+        return {
+            pointsDescriptionId: id + `-line${i + 1}-points`,
+            interceptDescriptionId: id + `-line${i + 1}-intercept`,
+            slopeDescriptionId: id + `-line${i + 1}-slope`,
+            pointsDescription: strings.srLinearSystemPoints({
+                lineSequence: i + 1,
+                point1X: srFormatNumber(line[0][0], locale),
+                point1Y: srFormatNumber(line[0][1], locale),
+                point2X: srFormatNumber(line[1][0], locale),
+                point2Y: srFormatNumber(line[1][1], locale),
+            }),
+            interceptDescription: getInterceptStringForLine(
+                line,
+                strings,
+                locale,
+            ),
+            slopeDescription: getSlopeStringForLine(line, strings),
+        };
+    });
+
     return (
-        <>
+        <g
+            aria-label={strings.srLinearSystemGraph}
```

Comment:
This aria-label is not getting read out when navigating to the whole graph. This is because the whole graph descriptions needs to happen at the mafs-graph.tsx level.